### DISCUSSION TITLE
Any plan to support 16kb page size on Android 15 release?

### DISCUSSION THREAD

**Discussion by Edison0716** [bytedance/android-inline-hook#69] [General]
https://developer.android.com/guide/practices/page-sizes?hl=zh-cn

**Comment by Colibrow** [comment]
> https://developer.android.com/guide/practices/page-sizes?hl=zh-cn

Thanks for reaching out! We're actually working on supporting a 16kb page size right now and plan to include it in the next release. Stay tuned!

**Reply by Edison0716** [reply]
@Colibrow 这个16kb的问题什么时候可以升？我这边先试试自己以16kb.编译一下

**Reply by Colibrow** [reply]
可以参考dev分支代码进行验证 https://github.com/bytedance/android-inline-hook/commit/9fedeee5d2216c1d431d6d1a441f83b87d2c54d5

**Comment by caikelun** [comment]
@Edison0716 https://github.com/bytedance/android-inline-hook/releases/tag/v1.0.10
